prac_06/languages.py

```diff
@@ -1,18 +1,25 @@
 """
 languages
-Estimate: 30 minutes
-Actual:   35 minutes
+Estimate: 20 minutes
+Actual:   25 minutes
+Store a list of programming languages and give back th dynamic typed ones.
```

Comment:
Everything's good. Just picked up on a minor typo. You typed: "th", missed the 'e'. Otherwise all good here.

Reply:
Thanks Eric for the review. I 've corrected the typo you mentioned.